relay-admin-dashboard/Cargo.toml

```
@@ -0,0 +1,19 @@
+[package]
+name = "relay-admin-dashboard"
```

```suggestion
name = "relay-dashboard"
```

To be honest I would keep the name simpler, just `relay-dashboard` and omit all the `admin` from any of the places.

---

Agreed!